CDCgov/prime-reportstream
# Onboarding ops/greene858/7284/receiver checks for new transport

This PR restores a section of code that was commented out for local testing. The code should be in place for production use. 

Test Steps:

Pull down entire branch
Do a Gradle clean and package
Run dev environment
Ensure that there is a receiver configured for REST transport, in this example we use oh-doh
check the receiver using the command line with: curl -X POST -H "client:simple_report" -d "" "http://localhost:7071/api/checkreceiver/org/oh-doh/receiver/elr"
confirm that the command returns JSON with "result": "success"
stop the REST mock container to simulate a disconnected receiver: docker-compose stop rest-webservice
check the receiver again with: curl -X POST -H "client:simple_report" -d "" "http://localhost:7071/api/checkreceiver/org/oh-doh/receiver/elr"
confirm that the command returns JSON with "result": "fail"
Ensure that there is a receiver configured for SFTP transport, example ak-phd
check this receiver with: curl -X POST -H "client:simple_report" -d "" "http://localhost:7071/api/checkreceiver/org/oh-doh/receiver/elr"
confirm that the command returns JSON with "result": "success"
stop the SFTP mock container to simulate a disconnected receiver: docker-compose stop sftp
check this receiver with: curl -X POST -H "client:simple_report" -d "" "http://localhost:7071/api/checkreceiver/org/oh-doh/receiver/elr"
confirm that the command returns JSON with "result": "fail"
run gradle testEnd2End
run integration test for RESTTransport
have a nice day
## Changes
- *Include a comprehensive list of changes in this PR*
- *(For web UI changes) Include screenshots/video of changes*

## Checklist

### Testing
